src/js/components/Tiles.js

```diff
@@ -196,13 +203,18 @@ export default class IndexTiles extends Component {
     }
 
     return (
-      <Tiles className={classes.join(' ')} onMore={onMore}
-        flush={this.props.flush} fill={this.props.fill}
-        selectable={this.props.onSelect ? true : false}
-        selected={selectionIndex}
-        size={this.props.size}>
-        {tiles}
-      </Tiles>
+      <div>
+        {actions && <Header size="small" justify="end" responsive={false} pad={{horizontal: 'small'}}>
```

Comment:
A very minor item. This seems like a bit too much javascript logic inside the JSX. I think we should move this outside the return().